### DISCUSSION TITLE
[Windows] How do I get access to the user's document folder?

### DISCUSSION THREAD

**Discussion by jcoder58** [microsoft/react-native-windows#13196] [Q&A] [answered]
Using  @dr.pogodin/react-native-fs I have the following code:

          const folderExists = await RNFS.exists(RNFS.ExternalDirectoryPath);

which throws a "Access is denied" exception.  `RNFS.ExternalDirectoryPath` is the path to the user's documents folder.  I've modified the app manifest to set capabilities to:

```
  <Capabilities>
    <Capability Name="internetClient" />
    <rescap:Capability Name="broadFileSystemAccess"/>
    <uap:Capability Name="removableStorage"/>
  </Capabilities>
```

Can anyone suggest what the correct settings are?

**Comment by chrisglein** [comment, marked as answer]
How are you using RNFS? My understanding is that Windows support for that hasn't been merged (see https://github.com/microsoft/react-native-windows/issues/2829).

According to these docs it appears that `broadFileSystemAccess` should be sufficient: https://learn.microsoft.com/en-us/windows/uwp/files/file-access-permissions#capabilities-for-accessing-other-locations

**Comment by mohdravish** [comment]
Hi everyone  i am developing a window app in react native 
I am using the @dr.pogodin/react-native-fs
But i wanted to open a any directory for eg:- C:\\Users\
of my local machine so how can i achieve  that any solution would be appreciated

**Reply by chrisglein** [reply]
By default React Native for Windows apps are UWP apps, and have restricted file permissions. Some reading:
- https://microsoft.github.io/react-native-windows/docs/native-code
- https://learn.microsoft.com/en-us/windows/uwp/files/file-access-permissions
